community/ecosystem-projects.md

```diff
@@ -22,11 +22,9 @@ The ecosystem project *must* enhance one of the existing core projects as oppose
 ### **Experimental Features**
 
 
-The ecosystem project may be an experimental implementation of a new feature that may eventually be merged into one of the core projects. A good example
-is https://github.com/argoproj-labs/applicationset project that simplifies managing Argo CD applications. The functionality is developed in
-a separate repository to increase development velocity and simplify onboarding early adopters.
+The ecosystem project may be an experimental implementation of a new feature that may eventually be merged into one of the core projects. Once the project is mature enough it may be considered for merging into the core project repository.
```

Comment:
```suggestion
The ecosystem project may be an experimental implementation of a new feature that may eventually be merged into one of the core projects. Once the project is mature enough it may be considered for merging into one of the core project repositories.
```